vegbilder-client/src/components/Header/Search/Search.tsx

```diff
@@ -286,6 +283,15 @@ const Search = ({ showMessage, setMapView }: ISearchProps) => {
     }
   };
 
+  //Enkelte fylker, f.eks. Troms of Finnmark, har flere navn (bla.a. på samisk) hvor alle navnene er satt sammen i en string av typen "navn - navn2 - navn3". Ettersom språket ellers er bokmål bruker vi bare dette for å få plass i nedtrekksmenyen.
+  const formatFylkesnavn = (fylke: string) => {
+    if (fylke.includes(" - ")) {
+      return fylke.split(" - ")[0];
```

Comment:
Foreslår følgande endring, fordi den handterer 0 til uendeleg mange mellomrom før og etter bindestrek (og andre lignande whitespace-teikn, det fins faktisk fleire typar mellomrom):
```suggestion
    if (fylke.includes("-")) {
      return fylke.split("-")[0].trim();
```

Reply:
Obs: Ser at eg har brukt kort bindestrek medan du har brukt ein lenger variant. Det skal sikkert vere den lange som du har brukt.

Reply:
Tror jeg skal ha brukt den vanlige -. Trykte ikke options + "-" i hvert fall 😄
Hadde med mellomrom fordi verdien fra api-et har det samme. Det vil si: eks. "Tromsø – nordsamisk – sørsamisk", men det har kanskje ikke noe å si?

Reply:
Eller, det er jo det trim() gjør, når jeg tenker meg om 😅